src/test/java/com/google/api/codegen/testdata/discoveries/ruby/ruby_compute.v1.json.baseline

```diff
@@ -29,9 +29,11 @@ items = service.fetch_all do |token|
   service.list_aggregated_addresses(project, page_token: token)
 end
 
-items.each do |name, addresses_scoped_list|
-  # TODO: Change code below to process each (name, addresses_scoped_list) pair:
-  puts String(name << " => " << addresses_scoped_list.to_json
+items.each do |item|
+  item.each do |name, addresses_scoped_list|
+    # TODO: Change code below to process each (name, addresses_scoped_list) pair:
+    puts String(name) + " => " + addresses_scoped_list.to_json
+  end
```

Comment:
I'm not sure if `page` is the right name either, the collection returned by `fetch_all` is the list of `items` pulled from the response, not the page. I think that `items` is still the correct name.

https://github.com/saicheems/toolkit/blob/master/src/test/java/com/google/api/codegen/testdata/discoveries/compute.v1.json#L199

Here's the discovery, the items field is what's collected by `fetch_all`.